os/arch/arm/src/armv7-r/arm_assert.c

```diff
@@ -374,12 +386,14 @@ static void unwind_backtrace_with_fp(arm_regs_t *regs, struct tcb_s *task)
 		if (unwind_frame_with_fp(&stack_frame, ustacksize) >= 0) {
 			/* Print the call stack address */
 #ifdef CONFIG_FS_ROMFS
-			char buffer[64];
-			get_symbol(current_addr, buffer);
-			lldbg("[<0x%p>] %s\n", (void *)current_addr, buffer);
-#else
-			lldbg("[<0x%p>]\n", (void *)current_addr);
+			char buffer[64] = { '\0' };
```

Comment:
I think get_symbol should verify NULL termination of buffer.
And if get_symbol verifies it, '\0' initialization is not needed.

Reply:
done